src/server/bin/plugin/pluginLogger.js

```diff
@@ -0,0 +1,41 @@
+﻿module.exports = function (silent) {
+  var previousLineEnded = true;
+  silent = !!silent;
+
+  function log(data, sameLine) {
+    if (silent) return;
+
+    if (!sameLine && !previousLineEnded) {
+      process.stdout.write('\n');
+    }
+
+    //if data is a stream, pipe it.
+    if (data.readable) {
+      data.pipe(process.stdout);
+      return;
+    }
+
+    if (!sameLine) data += '\n';
```

Comment:
I suggest either adding this in the call to `stdout.write(data + '\n')` or reassigning the data variable to a new string `data = data + '\n'` as this is modifying the data string which is owned by the caller.

Consider this:
```js
var okMsg = 'all is well';
function logOk() {
  logger.log(okMsg);
}

logOk();
logOk();
logOk();
```

This would output:
```
all is well
all is well

all is well


```